src/main/resources/application.yml

```diff
@@ -99,6 +99,6 @@ management:
 dpr:
   lib:
     definition:
-      locations: "reports/incident-report.json, reports/incident-count.json, reports/incident-with-people.json"
+      locations: "reports/incident-report.json, reports/incident-report-pecs.json, reports/incident-count.json, reports/incident-with-people.json"
```

Comment:
Some point might point the it at an endpoint in our service to load the definitions or use a database - but not yet